src/components/ServerConnections.js

```diff
@@ -40,10 +39,10 @@ class ServerConnections extends ConnectionManager {
 
         Events.on(this, 'localusersignedout', (_e, logoutInfo) => {
             setUserInfo(null, null);
+            // Reset cached views
+            document.dispatchEvent(new CustomEvent('skinunload', {}));
```

Comment:
I would remove the code related to that event and just call the `viewContainer` function directly.

```suggestion
            viewContainer.reset();
```